byte.h

```diff
@@ -7,8 +7,8 @@ extern unsigned int byte_chr();
 extern unsigned int byte_rchr();
 extern void byte_copy();
```

Comment:
For example, in `qmail-local.c`: `byte_copy(fnnewtph,3,"new");`

So:

```
#define byte_copy(d,n,s) (memcpy(d,s,n))
```

Or to preserve semantics:

```
#define byte_copy(d,n,s) (memmove(d,s,n))
```